config/samples/deployments/pvc-example.yaml

```diff
@@ -1,15 +1,13 @@
 kind: PersistentVolumeClaim
 apiVersion: v1
 metadata:
-  name: myclaim
   annotations:
     k8up.io/backup: 'true'
   namespace: myproject
```

Comment:
Is that correct? Do you intent to remove the `name` and not `namespace`?

Reply:
Yeah that's wrong...